contracts/contracts/strategies/BaseConvexMetaStrategy.sol

```diff
@@ -18,6 +18,7 @@ import { Helpers } from "../utils/Helpers.sol";
 abstract contract BaseConvexMetaStrategy is BaseCurveStrategy {
     using StableMath for uint256;
     using SafeERC20 for IERC20;
+    event MaxWithdrawalSlippageUpdated(uint256 _previousAmount, uint256 _newAmount);
```

Comment:
super nit: it's not really an amount but rather a percentage. so for code clarity could name these arguments something like `_prevMaxSlippagePercentage` and `_newMaxSlippagePercentage`.

Or just keep things brief since the event name is by itself very descriptive. like `_prev` and `_new`.

naming is hard :)

Reply:
tnx, changed